### PR TITLE
Add the netlink_event_tracker to handle netlink event

### PR DESCRIPTION
**Pull request by hwchiu**
- The Server support the netlink event tracker now.
- It will remove the veth from OVS after the container has been removed.
- Since some testing will change the ovs command file mode, we should force the parallel=1 to avoid testing fail.

### CODE REVIEW

File: netlink/netlink_handler.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package netlinkEvent
```
Comment (John-Lin) [inline netlink/netlink_handler.go]: package name seems wrong

File: netlink/netlink_handler_test.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
+package netlinkEvent
```
Comment (John-Lin) [inline netlink/netlink_handler_test.go]: package name seems wrong

**Comment by John-Lin** [comment]
Should consider the `netlink event tracker` only activate in unix socket server since we'll running two kinds of grpc servers

**Comment by hwchiu** [comment]
I will add a option to start the event track and we can decide which server should active that function from the arguments.

File: server/main.go [hunk 3]
```diff
@@ -25,10 +26,13 @@ type server struct {
 }
 
 func main() {
+
 	var tcpAddr string
 	var unixPath string
+	var nlEventTracker bool
 	flag.StringVar(&tcpAddr, "tcp", "", "Run as a TCP server and listen on target address")
 	flag.StringVar(&unixPath, "unix", "", "Run as a UNIX server and listen on target path")
+	flag.BoolVar(&nlEventTracker, "nl", false, "Run as a Netlink Event Tracker")
```
Comment (John-Lin) [inline server/main.go]: I would like to propose the flag names and usage descriptions
```go
flag.BoolVar(&nlEventTracker, "track-netlink", false, "Run as a netlink event tracker for garbage collection")

// OR

flag.BoolVar(&nlEventTracker, "netlink-gc", false, "Run as a netlink event tracker for garbage collection")
```
the command can be used like

`./network-controller -unix=/tmp/grpc.sock -track-netlink=true`

OR 

`./network-controller -unix=/tmp/grpc.sock -netlink-gc=true`

**Comment by zyfdegh** [comment]
Bool flags do not require users to specify true or false statement, to say, using `--track-netlink` is Okay which means true, not `-track-netlink=true`.  ;)

**Comment by hwchiu** [comment]
@zyfdegh  Thanks.
According to the manual. we need to avoid the usage `-xxx true` which is not support in bool form 
```
-flag
-flag=x
-flag x  // non-boolean flags only
```

**Pull request merged**